tools/make/golang.mk

```diff
@@ -20,36 +20,34 @@ go.build.%:
 	$(eval OS := $(word 1,$(subst _, ,$(PLATFORM))))
 	$(eval ARCH := $(word 2,$(subst _, ,$(PLATFORM))))
 	@echo "===========> Building binary $(COMMAND) with commit $(REV) in version $(VERSION) for $(OS) $(ARCH)"
-	@CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build -o $(OUTPUT_DIR)/$(OS)/$(ARCH)/$(COMMAND) $(ROOT_PACKAGE)/cmd/$(COMMAND)
+	CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build -o $(OUTPUT_DIR)/$(OS)/$(ARCH)/$(COMMAND) $(ROOT_PACKAGE)/cmd/$(COMMAND)
 
 # Build the envoy-gateway binaries in the hosted platforms.
 .PHONY: go.build
-go.build:
-	@$(MAKE) $(addprefix go.build., $(addprefix $(PLATFORM)., $(BINS)))
```

Comment:
guessing this got fixed using `SECONDARY` ? thanks for taking care of this

Reply:
This was actually addressed by moving the variable-setting to be before the imports in e953d5f58cb1f007018b1c61553e1df0fbf9b4d4

Reply:
ah thanks for pointing that out :)